deployment/distribution/src/main/kotlin/org/codice/ctk/Command.kt

```diff
@@ -110,18 +115,48 @@ private fun launchTests() {
 
     val summaryGeneratingListener = SummaryGeneratingListener()
     LauncherFactory.create().apply {
-        registerTestExecutionListeners(summaryGeneratingListener)
+        registerTestExecutionListeners(summaryGeneratingListener, TestNameListener())
     }.execute(request)
 
     PrintWriter(System.out).use { printer ->
+
         summaryGeneratingListener.summary.printFailuresTo(printer)
         summaryGeneratingListener.summary.printTo(printer)
 
         if (summaryGeneratingListener.summary.totalFailureCount > 0) {
-            System.out.println("TESTS FAILED")
+            System.out.println(ansi().fgRed().a("TESTS FAILED").reset())
             System.exit(1)
         }
 
-        printer.println("TESTS PASSED")
+        printer.println(ansi().fgGreen().a("TESTS PASSED").reset())
+    }
+}
+
+private class TestNameListener : TestExecutionListener {
+    override fun testPlanExecutionStarted(testPlan: TestPlan?) {
+        System.out.apply {
+            println()
+            println("----------------------------------")
+            println("SAML Conformance Test Kit Starting")
+            println("----------------------------------")
+        }
+    }
+
+    override fun executionFinished(testIdentifier: TestIdentifier?,
+                                   testExecutionResult: TestExecutionResult?) {
```

Comment:
❓ Is it possible to change `testIdentifier` and `testExecutionResult` to `TestIdentifier` and `TestExecutionResult` without the `?`? That way you don't need the `?.let` or the null check

Reply:
I was, in fact, able to do that! 🎉 I didn't know I could do that with a function override.